py/compile.c

```diff
@@ -2006,6 +2006,7 @@ STATIC void compile_expr_stmt(compiler_t *comp, mp_parse_node_struct_t *pns) {
                 case MP_TOKEN_DEL_DBL_SLASH_EQUAL: op = MP_BINARY_OP_INPLACE_FLOOR_DIVIDE; break;
                 case MP_TOKEN_DEL_SLASH_EQUAL: op = MP_BINARY_OP_INPLACE_TRUE_DIVIDE; break;
                 case MP_TOKEN_DEL_PERCENT_EQUAL: op = MP_BINARY_OP_INPLACE_MODULO; break;
+                case MP_TOKEN_DEL_AT_EQUAL: op = MP_BINARY_OP_INPLACE_MATMUL; break;
```

Comment:
It might be possible to compress this switch statement into a small array with a single byte lookup (if so it should be a separate PR to fully evaluate it).